tap_gorgias/streams.py

```diff
@@ -255,10 +259,80 @@ def sync(self, state, config):
             yield (self.stream, event)
         self.update_bookmark(state, max_synced_thru)
 
+class VoiceCallEvents(CursorStream):
+    name = 'voice_call_events'
+    replication_method = 'INCREMENTAL'
+    key_properties = ['id']
+    replication_key = 'created_datetime'
+    datetime_fields = set(['created_datetime'])
+    results_key = 'data'
+    url = '/api/phone/voice-call-events'
```

Comment:
voice_call_events are ascending order, so I don't think we can take advantage of any bookmarks/config start date? maybe we could use the cursor as a bookmark? I'm fine with how it is though. you mind including a comment that it's in ascending?